spec/controllers/admin/settings_controller_spec.rb

```diff
@@ -85,10 +85,16 @@
           end
         end
 
-        it "allows admins with policy_and_abuse role to update spam setting" do
-          put :update, params: { id: setting.id, admin_setting: { hide_spam: "1" } }
-          expect(setting.reload.hide_spam?).to be_truthy
-          it_redirects_to_with_notice(admin_settings_path, "Archive settings were successfully updated.")
+        {
+          hide_spam: true,
+          invite_from_queue_enabled: false,
+          invite_from_queue_number: 10
```

Comment:
10 is the default (`INVITE_FROM_QUEUE_NUMBER`), you should have this value be something different.